specification/containerservices/resource-manager/Microsoft.ContainerService/2017-07-01/containerService.json

```diff
@@ -0,0 +1,839 @@
+        "Standard_F2",
+        "Standard_F2s",
+        "Standard_F4",
+        "Standard_F4s",
+        "Standard_F8",
+        "Standard_F8s",
+        "Standard_G1",
+        "Standard_G2",
+        "Standard_G3",
+        "Standard_G4",
+        "Standard_G5",
+        "Standard_GS1",
+        "Standard_GS2",
+        "Standard_GS3",
+        "Standard_GS4",
+        "Standard_GS5",
+        "Standard_H16",
+        "Standard_H16m",
+        "Standard_H16mr",
+        "Standard_H16r",
+        "Standard_H8",
+        "Standard_H8m",
+        "Standard_L16s",
+        "Standard_L32s",
+        "Standard_L4s",
+        "Standard_L8s",
+        "Standard_M128s",
+        "Standard_M64ms",
+        "Standard_NC12",
+        "Standard_NC24",
+        "Standard_NC24r",
+        "Standard_NC6",
+        "Standard_NV12",
+        "Standard_NV24",
+        "Standard_NV6"
+      ]
+    },
+    "ContainerServiceCustomProfile": {
+      "properties": {
+        "orchestrator": {
+          "type": "string",
+          "description": "The name of the custom orchestrator to use."
+        }
+      },
+      "description": "Properties to configure a custom container service cluster.",
+      "required": [
+        "orchestrator"
+      ]
+    },
+    "KeyVaultSecretRef": {
+      "properties": {
+        "vaultID": {
+          "type": "string",
+          "description": "Key vault identifier."
+        },
+        "secretName": {
+          "type": "string",
+          "description": "The secret name."
+        },
+        "version": {
+          "type": "string",
+          "description": "The secret version."
+        }
+      },
+      "description": "Reference to a secret stored in Azure Key Vault.",
+      "required": [
+        "vaultID",
+        "secretName"
+      ]
+    },
+    "ContainerServiceServicePrincipalProfile": {
+      "properties": {
+        "clientId": {
+          "type": "string",
+          "description": "The ID for the service principal."
+        },
+        "secret": {
+          "type": "string",
+          "description": "The secret password associated with the service principal in plain text."
+        },
+        "keyVaultSecretRef": {
+          "$ref": "#/definitions/KeyVaultSecretRef",
+          "description": "Reference to a secret stored in Azure Key Vault."
+        }
+      },
+      "description": "Information about a service principal identity for the cluster to use for manipulating Azure APIs. Either secret or keyVaultSecretRef must be specified.",
+      "required": [
+        "clientId"
+      ]
+    },
+    "ContainerServiceOrchestratorProfile": {
+      "properties": {
+        "orchestratorType": {
+          "type": "string",
+          "enum": [
+            "Kubernetes",
+            "Swarm",
+            "DCOS",
+            "DockerCE",
+            "Custom"
+          ],
+          "x-ms-enum": {
+            "name": "ContainerServiceOrchestratorTypes",
+            "modelAsString": false
```

Comment:
Making sure you understand the risks here, setting `modelAsString` to `false` will cause breaking changes in sdks whenever the enum is modified. Please acknowledge.

Reply:
Hadn't thought of that, but good point. I don't see why we can't use `"modelAsString": true` here and elsewhere, so I'll make that change.